syntax = "proto3";

import "type.proto";
import "data_model_config.proto";
import "props.proto";

message DataObjectConfig {
    oneof config {
        BaseType base_config = 1;
        DataModelConfig data_model_config = 2;
    }
    Props props = 3;
}
